src/rust/ensogl/lib/core/src/animation/frp/animation/hysteretic.rs

```diff
@@ -0,0 +1,116 @@
+//! Animation that has a delayed onset.\
```

Comment:
typo at the end the line

Reply:
Deescribe it much more wordy here so people will understand that/ It would be cool to describe here also an example use case